boards/rpi-pico/Makefile.include

```diff
@@ -1,2 +1,7 @@
 CPU_MODEL       := RP2040
 PORT_LINUX      ?= /dev/ttyUSB0
+
+ifeq ($(PROGRAMMER),jlink)
+  JLINK_DEVICE = RP2040_M0_0
+  FLASHFILE = $(ELFFILE)
```

Comment:
```suggestion
```

I think it should even work like this, as the ELF file is the default flash file anyway. Please squash / amend directly.

Reply:
Hmmm, I share some doubts here. When I played with J-link before this PR, it defaults to `.bin` file and flashing doesn't work. (Because I didn't provide the flash memory offset I guess) so I add to pass `FLASHFILE` manually during the `make flash` command.
But maybe I'm wrong here. I'll try again tonight. I left my rpi-pico board at home.

Reply:
[This](https://github.com/RIOT-OS/RIOT/blob/718a4513b5d48ee6af407ea0b053507ad31369d6/makefiles/tools/jlink.inc.mk#L6) should be the culprit.

Reply:
OK, let's merge this as is. I can rebase https://github.com/RIOT-OS/RIOT/pull/19541 on top of that and drop this line again there, as then the ELF file would indeed be the default.

Reply:
As you wish. I've removed the waiting for PRs label.
I'll also try to test #19541 to see if there is any breakage.